testing/MLDB-1336-builtin-checks.py

```diff
@@ -0,0 +1,21 @@
+
+# This file is part of MLDB. Copyright 2016 Datacratic. All rights reserved.
+
+mldb = mldb_wrapper.wrap(mldb) # noqa
+
+import unittest
+
+class myTest(unittest.TestCase):
+
+    def test_sequence(self):
+        with self.assertRaisesRegexp(mldb_wrapper.ResponseException, "Executing builtin function exp: Can't convert value 'a' to double") as re:
```

Comment:
It's not critical to identify which argument, no, but it is important to let the user know that the problem was with an argument. Would it make sense to change the "executing builtin function" to "arguments to function" or can that message cover any number of execution problems which just happen to include argument errors?